src/dbus_mpris.rs

```diff
@@ -21,7 +21,7 @@ use librespot_core::{
     spotify_id::SpotifyAudioType,
```

Comment:
Before we can merge this, could you run `cargo fmt` and `cargo clippy`, so that CI is happy? :)